src/components/report/TextControls.css

```diff
@@ -26,63 +26,44 @@
   font-size: 0.85rem;
 }
 
-/* TODO: componentalizationreuse */
-.button {
-  padding: 0.25rem 0.5rem;
-  margin: 0;
-  border: 0;
-  background: none;
-  font-size: 0.85rem;
-  background: var(--unlighter-grey);
-  color: var(--dark-grey);
-  cursor: pointer;
-  transition: background 100ms;
-  user-select: none;
-  min-width: 2em;
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: 65%;
+
+.alignment-buttons {
+  display: flex;
+  align-items: flex-start;
 }
 
-.button:hover {
-  background-color: var(--unlighterer-grey);
+.alignment-buttons .button {
+  border-radius: 0;
+  min-width: 0;
 }
 
-.active {
-  background-color: var(--green);
+.button.left,
+.button.center,
+.button.right,
+.button.justify {
+  display: grid;
+  place-items: center;
+  padding: 0.325rem;
+  min-width: 1.75rem;
 }
 
-.initial.active {
-  background: var(--dark-grey);
-  color: white;
+.alignment-buttons svg {
+  width: 0.85rem;
+  height: 0.85rem;
+  fill: currentColor;
 }
 
-.alignment-buttons {
-  white-space: nowrap;
+.alignment-buttons span {
+  padding-top: 0.125rem;
```

Comment:
In logical properties that would be `padding-block-start`.

Reply:
I agree that that should be updated sometime, I still need to get into the habit of using start/end etc. I'm considering it low priority for now though, with all the other work.